qa/suites/fs/functional/tasks/snap-schedule.yaml

```diff
@@ -3,7 +3,7 @@ overrides:
     conf:
       mgr:
         debug mgr: 20
-        debug ms: 1
+        debug ms: 2
```

Comment:
If we are changing `qa/cephfs/conf` then why do we need these changes?

Reply:
I thought the `conf` will be overrode by this ?

Reply:
no, they are merged together

Reply:
Okay.